documentation/3.4.0/dockerized/docker-compose-project.yml

```diff
@@ -0,0 +1,231 @@
+version: '3'
```

Comment:
@MallanagoudaB dockerized  folder  should contain two docker file eventually. one for stand alone setup and one for survey&project setup. arrange them in subfolders if required